### PR TITLE
Update packages/x11 to latest 

### PR DESCRIPTION
**Pull request by heitbaum**
- fontconfig is a jump to devel. But have been running 2.13.92 for the last month.
- the next version 2.13.93 will be using meson build. (PRs in the devel releases 21.13.91,2.13.92 are mostly fixes) https://www.freedesktop.org/wiki/Software/fontconfig/Devel/
- fluxbox is only a change from .gz to .xz

all others are next increment packages

### CODE REVIEW

**Comment by lrusak** [comment]
be aware that xf86-input-libinput isn't built by default and requires explicit inclusion.

File: packages/x11/other/fontconfig/package.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="fontconfig"
-PKG_VERSION="2.13.1"
-PKG_SHA256="9f0d852b39d75fc655f9f53850eb32555394f36104a044bb2b2fc9e66dbbfa7f"
+PKG_VERSION="2.13.92"
```
Comment (lrusak) [inline packages/x11/other/fontconfig/package.mk:6]: please don't use devel packages unless explicitly required.

**Comment by heitbaum** [comment]
> be aware that xf86-input-libinput isn't built by default and requires explicit inclusion.

👍 building

```
$ PROJECT=Generic ARCH=x86_64 IMAGE_SUFFIX=rudi s/build xf86-input-libinput
BUILD      libinput (target) - 1.16.4
BUILD      xf86-input-libinput (target) - 0.30.0
```

File: packages/x11/other/fontconfig/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/x11/other/fontconfig/package.mk:6]: 👍 got it. Have dropped the commit.

**Review by lrusak — APPROVED** [review]
LGTM

**Review by lrusak — APPROVED** [review]
Just re-approving with the commit drop

**Pull request merged**